python/feathub/processors/flink/table_builder/sliding_window_utils.py

```diff
@@ -49,29 +46,17 @@ class SlidingWindowDescriptor:
     def __init__(
         self,
         step_size: timedelta,
-        limit: Optional[int],
         group_by_keys: Sequence[str],
-        filter_expr: Optional[str],
     ) -> None:
         self.step_size = step_size
-        self.limit = limit
         self.group_by_keys = group_by_keys
-        self.filter_expr = filter_expr
 
     @staticmethod
     def from_sliding_window_transform(
         sliding_window_agg: SlidingWindowTransform,
     ) -> "SlidingWindowDescriptor":
-        filter_expr = (
-            to_flink_sql_expr(sliding_window_agg.filter_expr)
```

Comment:
It seems that the feathub filter_expr instead of flink filter_expr is passed to java in this PR. Let's fix this bug and add a test for cases where these two filter_expr differs.